apps/shipments/templates/email/shipment_link.mjml

```diff
@@ -0,0 +1,64 @@
+{% load mjml %}
+{% block subject %}shipchain.io - {{username}} shared a shipment details page with you.{% endblock %}
+
+{% block html_body %}
+{% mjml %}
+    <mjml>
+      <mj-body background-color="#ec7143">
+          <mj-section>
+            <mj-column>
+
+              <mj-image src="https://profiles.shipchain.io/static/profiles/shipchain-logo-2.png"></mj-image>
+
+            </mj-column>
+          </mj-section>
+          <mj-section background-color="#FFFFFF" full-width="full-width">
+              <mj-column>
+                  <mj-divider border-color="#9C9C9C" border-width="1px" border-color="#7A7A7A"></mj-divider>
+                  <mj-text font-size="16px" color="#666666" font-family="helvetica">
+                    	<p>Hi,</p>
+                      <p>You're receiving this email because <a href="#" style="text-decoration:none">{{username}}</a> has granted you access to one of their shipments, which can be accessed using the URL below.</p>
+                    <br></br>
+                    <p><a href={{link}} style="text-decoration:none">{{link}}</a></p>
+                  </mj-text>
+                  <mj-text font-size="16px" color="#666666" font-family="helvetica">
+                      <p>Sincerely,</p>
+                    	<br></br>
+                      <p>The ShipChain team</p>
+                  </mj-text>
+                  <mj-divider border-color="#9C9C9C" border-width="1px" padding-top="60px" padding-bottom="0"></mj-divider>
+              </mj-column>
+          </mj-section>
+          <mj-section background-color="#FFFFFF" full-width="full-width" vertical-align="top" padding-top="0" padding-bottom="5px">
+              <mj-column vertical-align="top">
+                  <mj-text align="center" color="#666666" font-size="15px" padding-bottom="0">
+                      Stay Connected With Shipchain
+                  </mj-text>
+              </mj-column>
+          </mj-section>
+          <mj-section background-color="#FFFFFF" vertical-align="top" full-width="full-width" padding-top="0">
+            <mj-group>
+              <mj-column>
+                  <mj-image src="https://profiles.shipchain.io/static/profiles/twitter_orange.png" href="https://twitter.com/shipchain" width="40px"/>
```

Comment:
Transmission should be independent of profiles - these static assets should be hosted via transmission